File: c++/exercicios/sequencia/r.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main() {
  int A, B, C, N, VB, T;
  double PV, PA, PB, PC, PN, PVB;
  cin >> A >> B >> C >> N >> VB;
  T = A + B + C + N + VB;
  cout << fixed << setprecision(1);
  PV = (double)(A + B + C) / (double) T * 100;
  PA = ((double) A / (double) T) * 100;
  PB = ((double) B / (double) T) * 100;
  PC = ((double) C / (double) T) * 100;
  PN = ((double) N / (double) T) * 100;
  PVB = ((double) VB / (double) T) * 100;
  cout << "Total de votos: " << T << endl;
  cout << "Porcentagem válidos: " << PV << endl;
  cout << "Porcentagem A: " << PA << endl;
  cout << "Porcentagem B: " << B << endl;
  cout << "Porcentagem C: " << C << endl;
  cout << "Porcentagem nulos: " << PN << endl;
  cout << "Porcentagem brancos: " << PVB << endl;
  return 0;
} 

File: javascript/exercicios/sequencia/r.js
const A = 24;
const B = 20;
const C = 99;
const N = 30;
const VB = 86;

const T = A + B + C + N + VB;
const PV = ((A + B + C) / T) * 100;
const PA = (A / T) * 100;
const PB = (B / T) * 100;
const PC = (C / T) * 100;
const PN = (N / T) * 100;
const PVB = (VB / T) * 100;

console.log(T, PV.toFixed(1), PA.toFixed(1), PB.toFixed(1), PC.toFixed(1), PN.toFixed(1), PVB.toFixed(1));